src/client/__tests__/attributesDecoration.spec.js

```diff
@@ -0,0 +1,50 @@
+import tape from 'tape-catch';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+const proxyquireStrict = proxyquire.noCallThru();
+
+const inputValidation = sinon.stub();
+inputValidation.resolves({});
+
+const AttributesDecorationMockedClient = proxyquireStrict(
+  '../attributesDecoration',
+  {
+    './inputValidation': inputValidation
+  }
+).default;
+
+
+tape('ATTRIBUTES DECORATION / storage', assert => {
+
+  const client = AttributesDecorationMockedClient({});
+
+  client.setAttribute('attributeName1', 'attributeValue1');
+  client.setAttribute('attributeName2', 'attributeValue2');
+
+  let values = client.getAttributes();
+
+  assert.equal(values['attributeName1'], 'attributeValue1', 'It should be equal');
+  assert.equal(values['attributeName2'], 'attributeValue2', 'It should be equal');
+
+  client.removeAttribute('attributeName1');
+
+  assert.equal(client.getAttribute('attributeName1'), undefined, 'It should throw undefined');
+  assert.equal(client.getAttribute('attributeName2'), 'attributeValue2', 'It should be equal');
+
+  client.setAttributes({
```

Comment:
It would be nice to update `attributeName2` here with a different value that the initial one, to assert that an already existing attribute is properly updated.